inlineplz/interfaces/swarm.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+import requests
```

Comment:
requests isnt a builtin module so it should be in a 2nd import group https://www.python.org/dev/peps/pep-0008/#imports

Reply:
```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.
```